mods/ra/chrome/lobby-options.yaml

```diff
@@ -37,6 +37,12 @@ Background@LOBBY_OPTIONS_BIN:
 					Width: 225
 					Height: 20
 					Text: Build off Allies' ConYards
+				Checkbox@BOUNTY_CHECKBOX:
+					X: 170
+					Y: 70
+					Width: 225
+					Height: 20
+					Text: Bounty
```

Comment:
Can you please change this to "Kill Bounties" and swap the order with the `BUILDRADIUS_CHECKBOX` (so that the two ConYard options are aligned vertically?